modules/table/src/testFixtures/java/org/apache/ignite/internal/table/TxAbstractTest.java

```diff
@@ -477,6 +503,70 @@ private InternalTransaction deleteUpsertAll() {
         return tx;
     }
 
+    private void checkIfNull(Tuple res1, Function<Transaction, Tuple> retryFunc) {
+        if (res1 != null) {
+            return;
+        }
+
+        logger().error("Found null value after upsertAll.");
+
+        printDebugInfo();
+
+        var resExplicit = igniteTransactions.runInTransaction(retryFunc);
+
+        logger().info("Same explicit call resulted in {} value", resExplicit);
+
+        fail("Found null value after upsertAll.");
+    }
+
+    private void printDebugInfo() {
+        logger().info("Primary node for accounts table is [nodeName={}]", primaryNode(accounts));
+        logger().info("Cluster times:");
+        long latestTime = IgniteTestUtils.getFieldValue(txTestCluster.clientClock(), HybridClockImpl.class, "latestTime");
+        logger().info("Client clock [time={}]", HybridTimestamp.hybridTimestamp(latestTime));
+
+        txTestCluster.clocks().forEach((name, hybridClock) -> {
+                    long time = IgniteTestUtils.getFieldValue(hybridClock, HybridClockImpl.class, "latestTime");
+                    logger().info(
+                            "Cluster clock [cluster_name={}, time={}]",
```

Comment:
It's rather node_name and not the cluster one. Besides that we usually do not use underscore in variable names in logs. Anyway, who cares, it's a tmp log that'll be removed as soon as we will fix the problem.